src/styles/blocks/catalog.scss

```diff
@@ -17,6 +17,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-property: transform, color;
+    transition-duration: 300ms;
```

Comment:
I like how you solved it, looking better and shorter than solution in one line. Probably could save duration as variable, because you use it twice in code

Reply:
agree, better create a variable for duration, you want your transitions to be consistent throughout the site